README.md

```diff
@@ -1 +1,31 @@
-# Skill_api
\ No newline at end of file
+Openskill api
+
+openskill is a full REST API built with flask to manage an application to complete and standard data store for canonical and emerging skills, knowledge, abilities, tools, technolgies, and how they relate to jobs..
+It is the first test version developed for the BACKEND-DEVELOPMENT course taught by [REDI](https://www.redi-school.org/career-program-munich).
+
+
+
+<br>
+## Initialize database and migrations
+#### before run the app, please run below command
+
+    flask db init - this will create a folder inside the app directory.
+    
+    flask db migrate - this will create all the migrations.
```

Comment:
I don't see any migrations files in the submitted code. If you have used this to create migrations, they should be checked in to github so that other contributing to your repository can also run them.